code/modules/spacepods/construction.dm

```diff
@@ -0,0 +1,257 @@
+						"self_msg" = "You frabricate a pressure bulkhead for the {HOLDER}."
+					)
+				),
+				// 8. Bulkhead added
+				list(
+					"desc" = "A space pod with loose bulkhead panelling exposed.",
+					STATE_PREV = list(
+						"key"      = /obj/item/crowbar,
+						"vis_msg"  = "{USER} pops the {HOLDER}'s bulkhead panelling loose.",
+						"self_msg" = "You pop the {HOLDER}'s bulkhead panelling loose.",
+
+						"spawn"    = /obj/item/stack/sheet/metal,
+						"amount"   = 5,
+					),
+					STATE_NEXT = list(
+						"key"      = /obj/item/wrench,
+						"vis_msg"  = "{USER} secures the {HOLDER}'s bulkhead panelling.",
+						"self_msg" = "You secure the {HOLDER}'s bulkhead panelling."
+					)
+				),
+				// 9. Bulkhead secured with bolts
+				list(
+					"desc" = "A space pod with unwelded bulkhead panelling exposed.",
+					STATE_PREV = list(
+						"key"      = /obj/item/wrench,
+						"vis_msg"  = "{USER} unbolts the {HOLDER}'s bulkhead panelling.",
+						"self_msg" = "You unbolt the {HOLDER}'s bulkhead panelling."
+					),
+					STATE_NEXT = list(
+						"key"      = /obj/item/weldingtool,
+						"vis_msg"  = "{USER} seals the {HOLDER}'s bulkhead panelling with a weld.",
+						"self_msg" = "You seal the {HOLDER}'s bulkhead panelling with a weld."
+					)
+				),
+				// 10. Welded bulkhead
+				list(
+					"desc" = "A space pod with sealed bulkhead panelling exposed... It needs the armor now.",
+					STATE_PREV = list(
+						"key"      = /obj/item/weldingtool,
+						"vis_msg"  = "{USER} cuts the {HOLDER}'s bulkhead panelling loose.",
+						"self_msg" = "You cut the {HOLDER}'s bulkhead panelling loose."
+					),
+					STATE_NEXT = list(
+						"key"      = /obj/item/pod_parts/armor,
+						"vis_msg"  = "{USER} installs the {HOLDER}'s armor plating.",
+						"self_msg" = "You install the {HOLDER}'s armor plating.",
+						"delete"   = 1
+					)
+				),
+				// 11. Loose armor
+				list(
+					"desc" = "A space pod with unsecured armor.",
+					STATE_PREV = list(
+						"key"      = /obj/item/crowbar,
+						"vis_msg"  = "{USER} pries off {HOLDER}'s armor.",
+						"self_msg" = "You pry off {HOLDER}'s armor.",
+						"spawn"    = /obj/item/pod_parts/armor,
+						"amount"   = 1
+					),
+					STATE_NEXT = list(
+						"key"      = /obj/item/wrench,
+						"vis_msg"  = "{USER} bolts down the {HOLDER}'s armor.",
+						"self_msg" = "You bolt down the {HOLDER}'s armor."
+					)
+				),
+				// 12. Bolted-down armor
+				list(
+					"desc" = "A space pod with unwelded armor.",
+					STATE_PREV = list(
+						"key"      = /obj/item/wrench,
+						"vis_msg"  = "{USER} unsecures the {HOLDER}'s armor.",
+						"self_msg" = "You unsecure the {HOLDER}'s armor."
+					),
+					STATE_NEXT = list(
+						"key"      = /obj/item/weldingtool,
+						"vis_msg"  = "{USER} welds the {HOLDER}'s armor.",
+						"self_msg" = "You weld the {HOLDER}'s armor."
+					)
+				)
+				// EOF
+			)
+
+	spawn_result(mob/user as mob)
```

Comment:
NO RELATIVE PATHING

Reply:
we need the bot to detect relative pathing so it can tag the pr with it and everyone can laugh